INSTALL.md

```diff
@@ -219,42 +221,22 @@ for Windows Server.
 
 ### Troubleshooting Installation
 
-XXX a bunch of this is no longer relevant.
-
-Synapse requires pip 8 or later, so if your OS provides too old a version you
-may need to manually upgrade it::
-
-    sudo pip install --upgrade pip
-
-Installing may fail with `Could not find any downloads that satisfy the requirement pymacaroons-pynacl (from matrix-synapse==0.12.0)`.
-You can fix this by manually upgrading pip and virtualenv::
-
-    sudo pip install --upgrade virtualenv
-
-You can next rerun `virtualenv -p python3 synapse` to update the virtual env.
-
-Installing may fail during installing virtualenv with `InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.`
-You can fix this  by manually installing ndg-httpsclient::
-
-    pip install --upgrade ndg-httpsclient
+If `pip` crashes mid-installation for some reason (e.g. lost terminal), `pip` may
```

Comment:
Does this section even make sense to include? I don't think I've ever had pip crash on me...

Reply:
Yeah... I'm not so sure at this point. I haven't heard any troubleshooting questions about this particular problem and one can always just recreate the virtualenv.

I'll just switch it to direct people to Synapse Admins.